src/Moq/Guard.cs

```diff
@@ -204,7 +204,10 @@ public static void IsMockable(Type type)
 		{
 			if (!type.IsMockable())
 			{
-				throw new NotSupportedException(Resources.TypeNotMockable);
+				throw new NotSupportedException(
+					string.Format(
+						Resources.TypeNotMockable,
+						type.Name));
```

Comment:
For consistency with existing code, please use the `type.GetFormattedName()` extension method:

```suggestion
						type.GetFormattedName()));
```